README.md

```diff
@@ -16,10 +16,10 @@ where [BRIEFING_DATE] has the format "YYYYMMDD". This will create a new briefing
 Create the expected variable file in the folder with
 
 ```
-./wbcli.sh variables [BRIEFING_DATE] [FLIGHT_NUMBER] [LOCATION]
+./wbcli.sh variables [BRIEFING_DATE] [FLIGHT_CODE] [LOCATION] [SATTRACK_DATE]
```

Comment:
Very minor but: Write SATTRACKS_DATE (plural on sattracks) to be consistent with variable used in code.